g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/source/controller/TranslationCollectKeyAPI.java

```diff
@@ -67,6 +69,7 @@ public SourceAPIResponseDTO collectV1StringTranslation(
 		String newKey = StringUtils.isEmpty(sourceFormat) ? key
 				: (key + ConstantsChar.DOT + ConstantsChar.POUND + sourceFormat.toUpperCase());
 		String newSource = source == null ? "" : source;
+		logger.info("The parameters are: productName={}, version={}, component={}, locale={}, key={}, source={}", productName, version, component, locale, key, newSource);
```

Comment:
Do we need to log this in an interceptor?

Reply:
Not, just add the log content in controller